Source/Exercise02Tests/Contacts/ContactsFetcherTests.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  ContactsFetcherTests.swift
+//  Exercise02Tests
+//
+//  Created by Calebe Emerick on 16/02/2018.
+//  Copyright © 2018 Stone Pagamentos. All rights reserved.
+//
+
+@testable import Exercise02
+import Nimble
+import XCTest
+
+final class ContactsFetcherTests: XCTestCase {
+	
+	private let domain = "ContactsFetcherTests"
+	private let key = "ContactKey"
+	private var userDefaults: UserDefaults!
+	private var fetcher: ContactsFetcher!
+	
+	private let validContactList: [Contact] = [
+		Contact(name: "João da Silva", email: "joaosilva@gmail.com", phone: "21973459921", company: Company(name: "Coca Cola", cnpj: "22951114000150", activeSince: Date(), isMei: true)),
+		
+		Contact(name: "Maria Azevedo", email: "mariazev@hotmail.com", phone: "21998345602", company: Company(name: "Google", cnpj: "70128653000136", activeSince: Date(), isMei: false))
+	]
+	
+	override func setUp() {
+		super.setUp()
+		
+		userDefaults = UserDefaults(suiteName: domain)
+		cleanUserDefaults()
```

Comment:
Não é redundante chamar a limpeza no início de cada teste, sendo que ela já é chamada no `tearDown`?

Reply:
Sim, mas a limpeza no tearDown não estava funcionando, aí forcei sempre 😔

Reply:
Tendeu 😭